src/ocm/components/clusterConfiguration/StorageTable.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { ExpandComponentProps, AITableProps } from './StorageAITable';
+import { Host, WithTestID } from '../../../common';
+import { usePagination } from '../../../common/components/hosts/usePagination';
+import { StorageDetail } from './StorageDetail';
+import AITable from '../../../common/components/hosts/AITable';
+
+const getHostId = (host: Host) => host.id;
+
+export const DefaultExpandComponent: React.FC<ExpandComponentProps<Host>> = ({ obj }) => (
+  <StorageDetail key={obj.id} host={obj} />
+);
+
+type HostsTableProps = ReturnType<typeof usePagination> & {
```

Comment:
If we don't expose `onSelect`, we'd be passing `undefined` and then you don't need `isStorageTable`

```suggestion
type StorageTableProps = ReturnType<typeof usePagination> & {
  hosts: Host[];
  skipDisabled?: boolean;
  content: AITableProps<Host>['content'];
  actionResolver?: AITableProps<Host>['actionResolver'];
  children: React.ReactNode;
  selectedIDs?: AITableProps<Host>['selectedIDs'];
  setSelectedHostIDs?: AITableProps<Host>['setSelectedIDs'];
  ExpandComponent?: AITableProps<Host>['ExpandComponent'];
  className?: AITableProps<Host>['className'];
};
```